code/modules/reagents/newchem/drinks.dm

```diff
@@ -96,16 +97,34 @@
 /datum/reagent/ethanol/synthanol/on_mob_life(var/mob/living/M as mob, var/alien)
 
 	var/d = data
-
-	if(d >= collapse_start && prob(10))
-		M.emote("collapse")
-		var/datum/effect/system/spark_spread/s = new /datum/effect/system/spark_spread
-		s.set_up(3, 1, M)
-		s.start()
-	if(d >= braindamage_start && prob(33))
-		M.adjustBrainLoss(1)
+	if(M.isSynthetic()) //works normally on synthetics
+		if(d >= spark_start && prob(25))
+			var/datum/effect/system/spark_spread/s = new /datum/effect/system/spark_spread
+			s.set_up(3, 1, M)
+			s.start()
+		if(d >= collapse_start && prob(10))
+			M.emote("collapse")
+			var/datum/effect/system/spark_spread/s = new /datum/effect/system/spark_spread
+			s.set_up(3, 1, M)
+			s.start()
+		if(d >= braindamage_start && prob(33))
+			M.adjustBrainLoss(1)
+
+	else
+		holder.remove_reagent(id, 3.6) //gets removed from organics very fast
+		if(prob(25))
+			holder.remove_reagent(id, 15)
```

Comment:
My bad, I'd glanced over `remove_reagent` and not seen a safety net in there, I'd missed the call to 'update_total` though, and that's got the safety net
Sorry :blush: